generators/app/templates/.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+name: Release
```

Comment:
Actually, this is supposed to be the boilerplate for a web application, and not a library, so this file should not exist here